packit_service/worker/helpers/job_helper.py

```diff
@@ -60,6 +59,12 @@ def __init__(
         self._is_reporting_allowed: Optional[bool] = None
         self._is_gitlab_instance: Optional[bool] = None
 
+    def get_package_name(self):
```

Comment:
would it be too complicated to deduplicate this functionality?

Reply:
I can create a mixin, shared between helpers and handlers but I was wondering if this isn't too much for just a simple method. At the moment we have no mixins between helpers and handlers... Do we want to create the first one?